### PR TITLE
MvcAndTests

### PR DESCRIPTION
**Pull request by Yanetta**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -23,6 +26,26 @@
...
+                            <!--<webResources>-->
+                                <!--<resource>-->
+                                    <!--<directory>src/main/resources</directory>-->
+                                    <!--<targetPath>WEB-INF/classes</targetPath>-->
+                                <!--</resource>-->
+                                <!--<resource>-->
+                                    <!--<directory>extralibs</directory>-->
+                                    <!--<targetPath>WEB-INF/lib</targetPath>-->
+                                <!--</resource>-->
+                            <!--</webResources>-->
```
Comment (u2107680007911543774e7r) [inline pom.xml]: we don't push commented old code

File: src/main/java/workSpringData/controller/Temp.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+
+@RestController
+@RequestMapping("/temp")
+public class Temp {
```
Comment (u2107680007911543774e7r) [inline src/main/java/workSpringData/controller/Temp.java]: this is a test controller, we shouldn't push it

File: src/main/java/workSpringData/exeption/UpdateException.java [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package workSpringData.exeption;
```
Comment (u2107680007911543774e7r) [inline src/main/java/workSpringData/exeption/UpdateException.java]: package name mistype, should be **exception**

**Review by u2107680007911543774e7r — APPROVED** [review]
You have postman screenshots committed, but we need .json file collection export from postman

File: src/main/java/workSpringData/controller/CustomersController.java [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+    @GetMapping()
+    public @ResponseBody Set<Customers> getCustomers()  {
+        LOG.info(" in getCustomers");
+        Set<Customers> customers = customersService.getAllCustomers();
+        LOG.info("getCustomers finished");
+        return customers;
+    }
+
+    @GetMapping("/id/{id}")
```
Comment (SerhiiPasko) [inline src/main/java/workSpringData/controller/CustomersController.java]: You can set both of that varians to one request with params such as ?name=value

File: src/test/resources/schema.sql [hunk 5]
```diff
@@ -0,0 +1,17 @@
+CREATE SCHEMA IF NOT EXISTS MA_STUDENT;
+
+CREATE SEQUENCE IF NOT EXISTS CUSTOMERS_SEQ
```
Comment (SerhiiPasko) [inline src/test/resources/schema.sql]: You don't use this sequnce

**Review by SerhiiPasko — CHANGES_REQUESTED** [review]
Read my and Vadim's comments

**Review by SerhiiPasko — APPROVED** [review]
Good

**Pull request merged**